src/main/java/com/it4045/pokemonviewer/enterprise/dto/Weakness.java

```diff
@@ -0,0 +1,15 @@
+package com.it4045.pokemonviewer.enterprise.dto;
+
+public enum Weakness {
+    Rock,
+    Fire,
+    Water,
+    Dragon,
+    Grass,
+    Poison,
+    Flying,
+    Bug,
+    Electric,
+    Ground,
+    Steel
+}
```

Comment:
This is the enum that I created under dto for Weakness. I have listed few weaknesses here.